### DISCUSSION TITLE
Can I keep the filepreview image but suppress the actual file preview?

### DISCUSSION THREAD

**Discussion by boing0** [microsoft-search/pnp-modern-search#3448] [Q&A]
While I love the file preview functionality, I found a bug when accessing it via a SharePoint tab in New Teams. After around 5 previews you get a 400 error in the preview popup and you have to quit Teams and clear the app cache to get the search page back. This is only in New Teams, Classic Teams and obviously SPO are fine. I don't want to explain clearing cache to hundreds of users and I'm not holding my breath on Microsoft fixing Teams' embedded browser. So, is there a way to keep the preview popup but only show the preview image and not attempt to load the actual file preview? I deleted the data-preview-url from pnp-file-preview, but that kills the popup entirely.

The 400 error in the preview:
![image](https://github.com/microsoft-search/pnp-modern-search/assets/8752412/d842d3c5-6392-4442-ba54-75f35e734121)

Reload the tab and the 400 takes over the entire page:
![image](https://github.com/microsoft-search/pnp-modern-search/assets/8752412/33b4dca1-fee8-4bde-8598-7f7460666fbb)

**Comment by wobba** [comment]
Which version are you using? If version before v4.10 then I think this might be resolved with https://github.com/microsoft-search/pnp-modern-search/pull/3399

**Reply by boing0** [reply]
Thanks Mikael, I'm using 4.10.1. The 400 error on previews is only in New Teams and mirrors other issues I have with the embedded browser in Teams (in Files tab, editProps opens the wrong/cached doc properties panel unless you reload it).

I really appreciate all that you guys do, this tool is awesome!

**Reply by wobba** [reply]
Seems to be a generic Teams/SPFx issue then maybe if you see it with other experiences as well.

**Reply by boing0** [reply]
Thanks Mikael. Do you any contacts at MSFT for the folks who work on that part of Teams? I opened a standard support ticket but they haven't replied. Also opened a ticket over in SharePoint/sp-dev-docs since I'm using SP List Formatting to open the doc properties panel: https://github.com/SharePoint/sp-dev-docs/issues/9360

That bug is a killer for me, would love to reach the right group who works on Teams/SPFx. Thanks again.

**Reply by wobba** [reply]
@boing0 I would try to create a small SPFx repro and log the issue at [sp-dev-docs](https://github.com/SharePoint/sp-dev-docs).
